versioned_docs/version-2020fa/lecture2.md

```diff
@@ -251,7 +251,7 @@ Let's test these endpoints by running `yarn start`.
 
 Use your web browser to navigate to `localhost:8080/getSongs`. You should see the value of `songs`, the empty array `[]`, on the page.
 
-You can terminate the running of the script using <kbd>Ctrl</kbd> + <kbd>C</kbd> or <kbd>Cmd</kbd> + <kbd>C</kbd>.
+You can terminate the running of the script using <kbd>Ctrl</kbd> + <kbd>C</kbd>
```

Comment:
nit: add back the period.

Reply:
dang it I don't even get contributions from the extra commit because of squash and merge